tflite2onnx/op/resize.py

```diff
@@ -0,0 +1,128 @@
+import logging
+import tflite
+import numpy as np
+
+from tflite2onnx.layout import Layout
+from tflite2onnx import mapping
+from tflite2onnx.op.common import Operator
+
+logger = logging.getLogger('tflite2onnx')
+
+
+class Resize(Operator):
+    TypeMapping = {
+        tflite.BuiltinOperator.RESIZE_NEAREST_NEIGHBOR: 'Resize',
+        tflite.BuiltinOperator.RESIZE_BILINEAR: 'Resize',
+        # No RESIZE_BICUBIC in BuiltinOperator
+    }
+
+    def __init__(self, TFactory, index):
+        super().__init__(TFactory, index)
+        # Four choices:
+        # half_pixel, pytorch_half_pixel, align_corners, asymmetric, tf_crop_and_resize
+        self.attrs['coordinate_transformation_mode'] = 'half_pixel'
+        # This attribute is valid only if "mode" is "cubic".
+        # The coefficient 'a' used in cubic interpolation.
+        # Two common choice are -0.5 (in some cases of TensorFlow) and -0.75 (in PyTorch).
+        self.attrs['cubic_coeff_a'] = -0.75
+        self.attrs['exclude_outside'] = 0
+        self.attrs['extrapolation_value'] = 0.0
+        # Three interpolation modes: nearest (default), linear and cubic.
+        # The "linear" mode includes linear interpolation for 1D tensor
+        # and N-linear interpolation for N-D tensor
+        # (for example, bilinear interpolation for 2D tensor).
+        # The "cubic" mode includes cubic interpolation for 1D tensor
+        # and N-cubic interpolation for N-D tensor
+        # (for example, bicubic interpolation for 2D tensor).
+        self.attrs['mode'] = 'nearest'
+        # Four modes: round_prefer_floor (default, as known as round half down),
+        # round_prefer_ceil (as known as round half up), floor, ceil.
+        # Only used by nearest interpolation.
+        # It indicates how to get "nearest" pixel in input tensor from x_original,
+        # so this attribute is valid only if "mode" is "nearest".
+        self.attrs['nearest_mode'] = 'round_prefer_floor'
+
+        self.setInited()
+
+    @property
+    def type(self):
+        return 'Resize'
+
+    @property
+    def isRESIZE_BILINEAR(self):
+        op = self.tflite
+        opcode = self.model.OperatorCodes(op.OpcodeIndex()).BuiltinCode()
+        return opcode is tflite.BuiltinOperator.RESIZE_BILINEAR
+
+    @property
+    def isRESIZE_NEAREST_NEIGHBOR(self):
+        op = self.tflite
+        opcode = self.model.OperatorCodes(op.OpcodeIndex()).BuiltinCode()
+        return opcode is tflite.BuiltinOperator.RESIZE_NEAREST_NEIGHBOR
+
+    def parse(self):
+        logger.debug("Parsing %s...", self.type)
+        op = self.tflite
+        opcode = self.model.OperatorCodes(op.OpcodeIndex()).BuiltinCode()
+        assert(opcode is tflite.BuiltinOperator.RESIZE_BILINEAR
+               or tflite.BuiltinOperator.RESIZE_NEAREST_NEIGHBOR)
+
+        assert(op.InputsLength() >= 2), "Only first two arguments: image and size, are compulsory"
```

Comment:
I checked TFLite's kernel, both `resize_bilinear` and `resize_nearest_neighbor` require two inputs. Have we met scenarios against this?